<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(238, 238, 238);
background-color: rgb(255, 247, 239);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 222, 222);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font: bold;
color: rgb(255, 222, 222);
color: rgb(255, 151, 151);</string>
    </property>
    <property name="text">
     <string>Cat Care</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
font: bold;
color: rgb(255, 222, 222);
color: rgb(255, 151, 151);</string>
   </property>
   <property name="text">
    <string>Welcome to Cat Care!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MyCatLoad">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>My Cat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VaccinationLoad">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vaccination</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InventoryLoad">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
  </widget>
  <widget class="QLabel" name="bgPicLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>651</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="welcomeBGPic.qrc"/>
 </resources>
 <connections/>
</ui>
